sdk/storage/azure-storage-file/HISTORY.md

```diff
@@ -1,11 +1,17 @@
 # Change Log azure-storage-file
 
+## Version 12.0.0b5:
```

Comment:
I thought next version would be just `12.0.0`?

Reply:
We will have to do a final scrub to make sure the latest version is correct in the change logs. This is consistent with how at least key vault's HISTORY.md file specified the next version number.